example/services/nginx/nginx.conf

```diff
@@ -39,6 +39,26 @@ http {
             proxy_pass http://http:8080;
         }
 
+        location /http-with-location-and-rewrite {
```

Comment:
hmmm these changes look like they are in part me hacking on things in the example directory, rather than an intended example.

Reply:
I'll remove this auth and rewrite exploration that I was doing.

Reply:
> Maybe part of the mixed feeling is using http too much.
> The backend ends up being http://http:8080/http [insert yo dawg meme]

Yeah, there's a bit of a stutter. I'm going to leave it, though, because I don't want to think of a punchy-but-not-distracting name.